web/html/web-introduction/css-defined.md

```diff
@@ -21,7 +21,7 @@ aspects:
 
 In short, CSS is used to create responsive layouts, animations, styling text, and positioning containers. By applying CSS, you add the look and feel that you'd like to your websites.
 
-However, CSS is not good for inserting media, creating links, and it also doesn't allow math expressions and database storage.
+However, CSS is not good for inserting media, creating links, and it also doesn't allow database storage.
```

Comment:
The only thing I see that we can do for this particular insight is just this:
```suggestion

```

Reply:
We can remove the last sentence and the insight will still be a good entry-level insight

Reply:
And we also have to remove the question and create a new one. Something like:


???, which stands for ??? is a `stylesheet` language used to describe the presentation of an HTML, XML, or XHTML document. It's used to provide styling for text, images, & links.

* CSS
* Cascading Style Sheets
* cs
* Cascading Sheets

Reply:
these are all good @Stefan-Stojanovic :+1: apply them and request a review from me when ready